release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -369,6 +369,19 @@ Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable,
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-10-TLS-subject-alternative-names-required"]
+==== TLS X.509 certificates must have Subject Alternative Names
+
+X.509 certificates must have a properly set Subject Alternative Names field.
```

Comment:
The field name should be bold?
https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Reply:
No, I don't think so. This isn't a GUI field name. It's more of a key-value pair in a text file. The name Subject Alternative Name is a common name for this, but the actual key is something like `subjectAltName`.